test/libsolidity/smtCheckerTests/functions/this_external_getter_2.sol

```diff
@@ -1,12 +0,0 @@
-pragma experimental SMTChecker;
-
-contract C {
-	mapping (uint => uint) public map;
-
-	function f() public view {
-		uint y = this.map(2);
-		assert(y == map[2]); // This fails as false positive because of lack of support for external getters.
-	}
-}
-// ----
-// Warning 6328: (137-156): CHC: Assertion violation happens here.
```

Comment:
Why was this file removed?

Reply:
It was not removed, just renamed to `this_external_getter_mapping_1.sol ` and updated.
Unfortunately, the file is very small and the git evaluated the change to be more than 50%, so it could not store it as a rename, but it turned it into a deletion of the old file and creation of the new file.